<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_assurance</class>
 <widget class="QDialog" name="dialog_assurance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>711</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/b49ffd_500b234dccd544ec85f111c9702ef44e_mv2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>511</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Ajouter Assurance</string>
    </attribute>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>-5</y>
       <width>511</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/b49ffd_500b234dccd544ec85f111c9702ef44e_mv2.jpg);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;font color=white&gt; Nom du Compagnie</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;font color=white&gt; Type d'Assurance</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_type">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_assu">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;font color=white&gt; Prix (en DT)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_prix">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_compagnie">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>160</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;font color=white&gt; Reference :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_reference">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="mod_assu">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_11">
    <attribute name="title">
     <string>Supprimer Assurance</string>
    </attribute>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>-5</y>
       <width>511</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/b49ffd_500b234dccd544ec85f111c9702ef44e_mv2.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;font color=white&gt;Donner le Type de l'Assurance Ã  Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="supp_assu">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_18">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_12">
    <attribute name="title">
     <string>Afficher Assurance</string>
    </attribute>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/image/b49ffd_500b234dccd544ec85f111c9702ef44e_mv2.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="export_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Excel</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_assurance">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>511</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
